.github/ISSUE_TEMPLATE/server_installation_form.yaml

```
@@ -0,0 +1,88 @@
+name: Server installation issue
+description:  Report an issue with installing a server via nvim-lsp-installer
+labels: server-issue
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+          👋! This is not an issue template for questions! If you have questions, please refer to https://github.com/williamboman/nvim-lsp-installer/discussions/categories/q-a :)
+
+          Before filing an issue, make sure that you meet the minimum requirements mentioned in the README
+  - type: textarea
+    id: problem-description
+    attributes:
+      label: Problem description
+      description: A clear and concise description of what the issue is and why you think it's an issue with nvim-lsp-installer
+      validations:
+        required: true
+  - type: input
+    id: nvim-version
+    attributes:
+      label: Neovim version (>= 0.6)
+      description: Output of `nvim --version`
+      placeholder: |
+        NVIM v0.6.0-dev+209-g0603eba6e
+        Build type: Release
+        LuaJIT 2.1.0-beta3
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: nvim-lsp-installer version
+      description: commit hash, `git log --pretty=%h -1`
+      placeholder: 284e914
+  - type: input
+    attributes:
+      label: nvim-lspconfig version
+      description: commit hash, `git log --pretty=%h -1`
+      placeholder: 299e6395
```

I think these are a bit overkill. What about a simple checkbox? (I'm not sure if the following is valid, need to find the schema for this file)

```suggestion
  - type: checkbox
    attributes:
      label: I've recently downloaded the latest plugin versions
      description: Make sure you've downloaded the latest plugin versions.
```